apps/full-site-editing/full-site-editing-plugin/blocks/post-content/index.php

```diff
@@ -1,11 +1,37 @@
 <?php
 
 function render_post_content_block( $attributes, $content ) {
+	// Early return to avoid infinite loops in the REST API
+	if ( is_admin() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
```

Comment:
Out of curiosity what does the trace look like? Are there similar rendering conditions we need to be aware of?